src/main/java/com/hotels/service/tracing/zipkintohaystack/forwarders/zipkin/http/ZipkinForwarder.java

```diff
@@ -37,18 +39,33 @@
 
     private final OkHttpSender sender;
 
-    public ZipkinForwarder(String endpoint, int maxInFlightRequests, int writeTimeoutMillis, boolean compressionEnabled, int maxIdleConnections) {
-        OkHttpSender.Builder builder = OkHttpSender.newBuilder()
-                .endpoint(endpoint)
-                .maxRequests(maxInFlightRequests)
-                .writeTimeout(writeTimeoutMillis)
-                .compressionEnabled(compressionEnabled);
+    public ZipkinForwarder(String endpoint, int maxInFlightRequests, int writeTimeoutMillis, boolean compressionEnabled, int maxIdleConnections, boolean ignoreSslErrors) {
+        try {
+            OkHttpSender.Builder builder = OkHttpSender.newBuilder()
+                    .endpoint(endpoint)
+                    .maxRequests(maxInFlightRequests)
+                    .writeTimeout(writeTimeoutMillis)
+                    .compressionEnabled(compressionEnabled);
 
-        builder.clientBuilder()
-                .connectionPool(new ConnectionPool(maxIdleConnections, 5, TimeUnit.MINUTES))
-                .pingInterval(60, TimeUnit.SECONDS);
+            builder.clientBuilder()
+                    .connectionPool(new ConnectionPool(maxIdleConnections, 5, TimeUnit.MINUTES))
+                    .pingInterval(61, TimeUnit.SECONDS);
```

Comment:
Is there a reason behind the `pingInterval` change?

Reply:
No, this was done accidentally. Thanks for catching.